static/src/stylesheets/module/_adslot.scss

```diff
@@ -454,35 +454,38 @@
         }
     }
 }
-.ad-slot--paid-for-badge--gallery {
-    border-top: 1px dotted $media-mute;
-    border-bottom: 0;
-    padding-top: $gs-baseline / 3;
 
-    @include mq(desktop) {
-        width: gs-span(3);
-        position: absolute;
-    }
+.content--media:not(.paid-content--advertisement-feature) {
+    .ad-slot--paid-for-badge--gallery {
+        border-top: 1px dotted $media-mute;
+        border-bottom: 0;
+        padding-top: $gs-baseline / 3;
 
-    .content__main-column > & {
         @include mq(desktop) {
-            top: 100%;
+            width: gs-span(3);
+            position: absolute;
         }
 
-        @include mq(leftCol) {
-            left: -160px;
-        }
+        .content__main-column > & {
+            @include mq(desktop) {
+                top: 100%;
+            }
 
-        @include mq(wide) {
-            left: -240px;
-        }
-    }
+            @include mq(leftCol) {
+                left: -160px;
+            }
 
+            @include mq(wide) {
+                left: -240px;
```

Comment:
Shouldn't these reference `-(gs-span(3) + $gs-gutter)`?

I know you're only indenting but while you're here...